src/pages/signup/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { useAccount } from "wagmi";
+import Link from "next/link";
+
+import { Layout } from "~/layouts/DefaultLayout";
+import { config } from "~/config";
+import { ConnectButton } from "~/components/ConnectButton";
+import { JoinButton } from "~/components/JoinButton";
+import { Info } from "~/components/Info";
+import { EligibilityDialog } from "~/components/EligibilityDialog";
+import { useMaci } from "~/contexts/Maci";
+import { Chip } from "~/components/ui/Chip";
+
+export default function SignupPage() {
+  const { isConnected } = useAccount();
+  const { isRegistered } = useMaci();
+
+  return (
+    <Layout>
+      <EligibilityDialog />
+
+      <div className="flex flex-col items-center justify-center gap-4">
+        <h1 className="max-w-screen-lg text-center font-mono">
+          {config.eventName.toUpperCase()}
+        </h1>
+        <h2 className="max-w-screen-lg text-center font-mono">
+          {config.roundId.toUpperCase()}
+        </h2>
+        <p className="max-w-screen-md text-center text-xl">
+          {config.startsAt.toLocaleDateString("en-US", {
+            year: "numeric",
+            month: "long",
+            day: "numeric",
+          })}{" "}
+          -{" "}
```

Comment:
Spacing should be done with styles

Reply:
how about now that using flex, span, and gap?

Reply:
You can just use margin style prop